terraform/cross-account-IAM/iam-prisoner-money.tf

```diff
@@ -0,0 +1,61 @@
+# generated by https://github.com/ministryofjustice/money-to-prisoners-deploy
+# IAM roles and policies used by Prisoner Money team's namespaces
+
+data "aws_iam_policy_document" "prisoner-money-kiam-trust-chain" {
+  # KIAM trust chain to allow pods to assume roles defined below
+  statement {
+    principals {
+      type        = "AWS"
+      identifiers = ["arn:aws:iam::754256621582:role/nodes.live-1.cloud-platform.service.justice.gov.uk"]
+    }
+    actions = ["sts:AssumeRole"]
+  }
+}
+
+variable "money-to-prisoners-test-tags" {
+  type = map(string)
+  default = {
+    business-unit    = "HMPPS"
+    application      = "money-to-prisoners"
+    is-production    = "false"
+    environment-name = "test"
+    owner            = "prisoner-money"
+  }
```

Comment:
perhaps you also might want `infrastructure-support = var.infrastructure-support` as a line here